ion/binarywriter_test.go

```diff
@@ -175,7 +175,7 @@ func TestWriteBinaryTimestamp(t *testing.T) {
 
 func TestWriteBinaryDecimal(t *testing.T) {
 	eval := []byte{
-		0x50,       // 0.
```

Comment:
Might want to leave this in for 0. since that seems to be specified in the spec?

Reply:
Yes, that's right. Missed that in the spec and you explained it well! Thank you!